README.md

```diff
@@ -11,25 +12,38 @@ ITKElastix
 Overview
 --------
 
-Provides an [ITK](https://www.itk.org) Python interface to [elastix](https://elastix.lumc.nl/), a toolbox for rigid and nonrigid registration of images.
+Provides an [ITK](https://www.itk.org) Python, JavaScript, and WebAssembly interface to [elastix](https://elastix.lumc.nl/), a toolbox for rigid and nonrigid registration of images.
 
 elastix is open source software, based on the well-known [Insight Toolkit (ITK)](https://discourse.itk.org). The software consists of a collection of algorithms that are commonly used to solve (medical) image registration problems. The modular design of elastix allows the user to quickly configure, test, and compare different registration methods for a specific application.
 
+[👨‍💻 **Live JavaScript API Demo** ✨](https://itk-wasm-elastix-app-js.on.fleek.co/ ':include :type=iframe width=100% height=800px')
```

Comment:
This is really neat! Great demonstration, very accessible.

At a glance I see a handful of (non-critical) tweaks that could be made. Should issues with `itk-wasm-elastix-app-js.on.fleek.co` be entered in the ITKElastix repository or elsewhere?

Reply:
Yes, please add issues to this repository, thanks!